Social Distancing Measures.ipynb

```diff
@@ -0,0 +1,386 @@
+{
```

Comment:
<p>Use the absolute scale please?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NirantK/coronaIndia/pull/26/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>I had used the default graph provided in the SEIRSPLUS library. I have removed this graph altogether in the new commit.	</p><p> </p>